mage_ai/frontend/components/Backfills/Table/index.tsx

```diff
@@ -124,7 +124,7 @@ function BackfillsTable({
             key="started_at"
             monospace
             small
-            title={startedAt ? `UTC: ${startedAt.slice(0, 19)}` : null}
+            title={startedAt ? utcStringToElapsedTime(startedAt.slice(0, 19)) : null}
```

Comment:
@anniexcheng We can remove the `.slice(0, 19)` from here since making the date displayed more concise is no longer a concern (it was to trim off the microseconds).